Emby.Server.Implementations/HttpServer/ResponseFilter.cs

```diff
@@ -32,10 +37,16 @@ public ResponseFilter(ILogger logger)
         /// <param name="dto">The dto.</param>
         public void FilterResponse(IRequest req, HttpResponse res, object dto)
         {
+            foreach(KeyValuePair<string, string> header in _server.GetCorsHeaders(req))
+            {
+                res.Headers.Add(header.Key, header.Value);
+            }
             // Try to prevent compatibility view
-            res.Headers.Add("Access-Control-Allow-Headers", "Accept, Accept-Language, Authorization, Cache-Control, Content-Disposition, Content-Encoding, Content-Language, Content-Length, Content-MD5, Content-Range, Content-Type, Date, Host, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since, Origin, OriginToken, Pragma, Range, Slug, Transfer-Encoding, Want-Digest, X-MediaBrowser-Token, X-Emby-Authorization");
-            res.Headers.Add("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE, PATCH, OPTIONS");
-            res.Headers.Add("Access-Control-Allow-Origin", "*");
+            res.Headers["Access-Control-Allow-Headers"] = ("Accept, Accept-Language, Authorization, Cache-Control, " +
```

Comment:
why is it defined here instead of being defined in a parameter-controlled fashion in `GetCorsHeaders()`?

Reply:
My intent was to provide the default headers in `GetCorsHeaders` and the caller can then override the return value with whatever they need. I think this is more flexible for any future usage. I see there is a pull request for adding CORS to exception so I expect this function call to be added there. Should I rename the function to `GetDefaultCorsHeaders`?